backend/src/agents/datastore_agent.py

```diff
@@ -106,7 +104,11 @@ async def get_semantic_layer_cache(llm, model):
 
 @chat_agent(
     name="DatastoreAgent",
-    description="This agent is responsible for handling database queries to the bloomberg.csv dataset. This includes retrieving ESG scores, financial metrics, and other bloomberg-specific information. It interacts with the graph database to extract, process, and return ESG-related information from various sources, such as company sustainability reports or fund portfolios. This agent can not complete any task that is not specifically about the bloomberg.csv dataset.",  # noqa: E501
+    description="This agent is responsible for answering questions about the bloomberg.csv dataset. This includes "
+                "retrieving ESG scores, financial metrics, and other bloomberg-specific information. It interacts with "
+                "the graph database to extract, process, and return ESG-related information from various sources, such "
+                "as company sustainability reports or fund portfolios. This agent can not answer questions that do not "
```

Comment:
I know that is was like this before
but I've heard that positive prompt engeneering works better than negative so I wonder if 
"This agent can only answer questions that specifically reference the bloomberg.csv dataset." would work better - I'll try it out

Reply:
didn't make a difference to the question I was asking
which was
What was the ESG score of AstraZeneca in 2020?
the answer I got was stil data store agent
{ "agent": "DatastoreAgent", "tool": "generate cypher query", "parameters": { "question_intent": "retrieve", "operation": "select", "question_params": "AstraZeneca, ESG score", "aggregation": "none", "sort_order": "none", "timeframe": "2020" }, "reasoning": "The question asks for the ESG score of AstraZeneca in 2020, which is specific data that is likely to be stored in the bloomberg.csv dataset. The DatastoreAgent is designed for querying this dataset, making it the appropriate choice for retrieving historical ESG scores. The 'generate cypher query' tool will allow us to form the necessary query to extract this data." }